tests/test-cases/Group6-VIC-Machine/6-02-Default.robot

```diff
@@ -16,10 +16,82 @@
+    Should Contain  ${ret}  vic-machine-linux delete failed
+    Should Not Contain  ${ret}  panic:
+
+    ${ret}=  Run  bin/vic-machine-linux inspect --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+    Should Contain  ${ret}  vic-machine-linux inspect  failed
+    Should Not Contain  ${ret}  panic:
+
+    ${ret}=  Run  bin/vic-machine-linux ls --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+    Should Contain  ${ret}  vic-machine-linux ls failed
+    Should Not Contain  ${ret}  panic:
+
+    ${ret}=  Run  bin/vic-machine-linux upgrade --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+    Should Contain  ${ret}  vic-machine-linux upgrade failed
+    Should Not Contain  ${ret}  panic:
+
+    ${ret}=  Run  bin/vic-machine-linux configure --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=INCORRECT
+    Should Contain  ${ret}  vic-machine-linux configure failed
+    Should Not Contain  ${ret}  panic:
+
+Check That VMOMI Sessions Don't Leak From VIC Machine
+    Set Test Environment Variables
+    ${output}=  Run  bin/vic-machine-linux ls --target %{TEST_URL} --debug=1 --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD}
+    Log  ${output}
+    ${thumbprint}=  Get Thumbprint From Log  ${output}
+    Should Not Have VMOMI Session  ${thumbprint}
+
+    ${output}=  Install VIC Appliance To Test Server
+    Log  ${output}
+    ${thumbprint}=  Get Thumbprint From Log  ${output}
+    Should Not Have VMOMI Session  ${thumbprint}
+
+    ${output}=  Run  bin/vic-machine-linux inspect --target %{TEST_URL} --debug=1 --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --name=%{VCH-NAME}
+    Log  ${output}
+    ${thumbprint}=  Get Thumbprint From Log  ${output}
+    Should Not Have VMOMI Session  ${thumbprint}
+
+    ${output}=  Run  bin/vic-machine-linux upgrade --target %{TEST_URL} --debug=1 --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD}
+    # upgrade fails since but that's ok for this test -- it still creates a session before failing
+    Log  ${output}
+    ${thumbprint}=  Get Thumbprint From Log  ${output}
+    Should Not Have VMOMI Session  ${thumbprint}
+
+    ${output}=  Run  bin/vic-machine-linux configure --target %{TEST_URL} --debug=1 --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --name=%{VCH-NAME}
+    Log  ${output}
+    ${thumbprint}=  Get Thumbprint From Log  ${output}
+    Should Not Have VMOMI Session  ${thumbprint}
+
+    ${output}=  Run  bin/vic-machine-linux delete --target %{TEST_URL} --debug=1 --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --name=%{VCH-NAME}
```

Comment:
This appears to be leaking network artifacts (`VCH-12949-9989-bridge` in logs from build https://ci.vcna.io/vmware/vic/12949). I'd suggest using `Cleanup VIC Appliance On Test Server` https://github.com/vmware/vic/blob/master/tests/resources/VCH-Util.robot#L275 instead since it cleans up the bridge network as well.